doc/performance-investigations/0008-use-gthread-grpc/summary.md

```diff
@@ -0,0 +1,37 @@
+# Title
+
+Gevent is currently in experimental status within the Google client libraries; the impact of switching to gthread as a gunicorn worker type has been investigated, as this is formally supported. Additionally, the gRPC protocal as an alternative to http has been investigated. This cannot currently be used with the gevent worker type.
```

Comment:
I think this needs to be made clearer. Gevent is in experimental status in the Google gRPC client libraries. We have not had an issue with this until now as we have been using Datastore over HTTP (via requests). We uncovered the issue as the pubsub library doesn't have a HTTP option and must be used over gRPC.